translator/translate/logs/logs_collected/files/collect_list/ruleTimestampFormat.go

```diff
@@ -156,6 +160,9 @@ func (t *TimestampRegax) ApplyRule(input interface{}) (returnKey string, returnV
 		res = "(" + res + ")"
 		returnKey = "timestamp_regex"
 		returnVal = res
+		if _, err := regexp.Compile(res); err != nil {
+			translator.AddErrorMessages(GetCurPath()+"timestamp_format", fmt.Sprintf("Timestamp format %s is invalid", val))
+		}
```

Comment:
Is there a unit test for this? Sorry just caught it

Reply:
I did think about this before pushing out this PR but there was no `ruleTimestampFormat_test.go` at the moment. I can create one though

Reply:
That would be great. Gotta increase code coverage where it's easy 😃  otherwise LGTM so once you add that you can DM me and I'll approve if I don't see the email notification first